### PR TITLE
Feature/802

### PR DESCRIPTION
**Pull request by MathiasNielsen1206**
# Description

Notification Pop-up for lost connectivity

Fixes #\802

## Type of change

- [x] New feature (non-breaking change which adds functionality)

**Development Configuration**
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation, if necessary
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works, if necessary
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have Acceptance Tested this on an iOS device.

### CODE REVIEW

**Review by Sijmon — DISMISSED** [review]
Looks good champ

File: lib/main.dart [hunk 1]
```diff
@@ -37,10 +45,21 @@ void _runApp() {
...
+              .where((bool currentState) =>
+                      lastState != currentState || firstTimeLogIn),
           builder: (BuildContext context, AsyncSnapshot<bool> snapshot) {
             lastState = snapshot.data;
-            first = false;
+            //To make sure we only listen to the stream once we take advantage
+            // of firstTimeLogin bool value
+            if(firstTimeLogIn== true){
+              _api.connectivity.connectivityStream.listen((dynamic event) {
+                print(event);
```
Comment (JakobFaarGreg) [inline lib/main.dart]:
```suggestion
```

File: lib/blocs/auth_bloc.dart [hunk 2]
```diff
@@ -51,6 +51,7 @@ class AuthBloc extends BlocBase {
       if (status) {
         // Get the role of a specific user
         _api.user.role(username).listen((int role) async {
+
```
Comment (JakobFaarGreg) [inline lib/blocs/auth_bloc.dart]:
```suggestion
```

**Comment by Brandhoej** [comment]
Okay :t-rex:

**Review by Brandhoej — APPROVED** [review]

**Review by JakobFaarGreg — APPROVED** [review]

**Pull request merged**